man/bundle-install.ronn

```diff
@@ -99,33 +54,16 @@ time `bundle install` is run, use `bundle config` (see bundle-config(1)).
   appropriate platform-specific gem exists on `rubygems.org` it will not be
   found.
 
-* `--no-cache`:
-  Do not update the cache in `vendor/cache` with the newly bundled gems. This
-  does not remove any gems in the cache but keeps the newly bundled gems from
-  being cached during the install.
-
-* `--no-prune`:
-  Don't remove stale gems from the cache when the installation finishes.
-
-* `--path=<path>`:
-  The location to install the specified gems to. This defaults to Rubygems'
-  setting. Bundler shares this location with Rubygems, `gem install ...` will
-  have gem installed there, too. Therefore, gems installed without a
-  `--path ...` setting will show up by calling `gem list`. Accordingly, gems
-  installed to other locations will not get listed.
-
 * `--quiet`:
   Do not print progress information to the standard output. Instead, Bundler
   will exit using a status code (`$?`).
 
 * `--retry=[<number>]`:
   Retry failed network or git requests for <number> times.
 
-* `--shebang=<ruby-executable>`:
-  Uses the specified ruby executable (usually `ruby`) to execute the scripts
-  created with `--binstubs`. In addition, if you use `--binstubs` together with
-  `--shebang jruby` these executables will be changed to execute `jruby`
-  instead.
+* `--redownload`:
+  Force download every gem, even if the required versions are already available
```

Comment:
Force downloads every gem (add 's' after 'download'), otherwise looks good to me!